config/30-votingworks.conf

```diff
@@ -9,7 +9,59 @@ template(name="outfmt" type="list" option.jsonf="on") {
  }
 
 # reformat json messages into one json blob and send to vx-logs.log
-if $syslogtag == "votingworksapp:" and re_match($msg,'^ *\\{.*\\} *$') then /var/log/vx-logs.log;outfmt
+if $syslogtag == "votingworksapp:" and re_match($msg,'^ *\\{.*\\} *$') then /var/log/vx-logs.log;jsonfmt
```

Comment:
small preference for using `vxapp` instead of the lengthier `votingworksapp`, but not necessary to fix here, maybe something to consider separately?